### ISSUE TITLE
Can't call method "exe" on an undefined value

### ISSUE THREAD

**Issue by eserte** [Perl5-Alien/Alien-gmake#1]
If following the instructions in https://metacpan.org/pod/distribution/Alien-Base/lib/Alien/Base/FAQ.pod#Gnu-Makefiles to use `Alien::gmake` in a `Build.PL` I get the error:
```
Can't call method "exe" on an undefined value at /usr/perl5.20.2p/lib/site_perl/5.20.2/Alien/gmake.pm line 39.
```
It looks like internally something like this is called:
```
$ perl -MAlien::gmake -e 'warn Alien::gmake::alien_helper->{gmake}->()'              
Can't call method "exe" on an undefined value at /usr/perl5.20.2p/lib/site_perl/5.20.2/Alien/gmake.pm line 39.
```
It would work if it was:
```
$ perl -MAlien::gmake -e 'warn Alien::gmake::alien_helper->{gmake}->("Alien::gmake")'
gmake at -e line 1.
```

**Comment by plicease** [comment]
This should be fixed in 0.11.  Thank you for reporting it.

**Comment by plicease** [comment]
I am also updating the FAQ to require 0.11.
